reference/clever-tools/networkgroups.md

```diff
@@ -0,0 +1,192 @@
+
+```sh
+clever ng --ng 'first-ng' members list
+```
+
+You should get something like:
+
+```log
+Member ID                                 Member Type                Label                                     Domain Name
+───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+app_d60ef23d-80d7-4095-b8d7-7f42bea137ea  application                My app 1                                  my-app-1
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- [`--json` or `-j` shows output as JSON](#--json-or--j-shows-output-as-json)
+
+### Add an external node category
+
+External node categories are <TODO>. You can have plenty of them, as it helps keeping this organized.
+
+Adding an external node category is very similar to the example in ["Add a member"](#add-a-member), except you'll use a custom member ID and `'external'` for the `--type`.
+
+Here is an example:
+
+```sh
+clever ng --ng 'first-ng' members add --member-id 'dev-laptops-category' --type 'external' --domain-name 'dev-laptops' --label 'Developer laptops'
+```
+
+> I voluntarily used `--member-id` and `--domain-name` but you can have similar ones if you want.
+
+```log
+Successfully added member 'dev-laptops-category' to networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tip:** [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+
+### Join a networkgroup
+
+To join a networkgroup, you need to [add an external node category](#add-an-external-node-category) and run `sudo clever ng join`. This command uses `wg-quick` under the hood, which needs privileges to create network interfaces. That's why you need to use `sudo`.
+
+> Clever Cloud's networkgroups use WireGuard®. Therefore, this command requires WireGuard® commands available on your computer.
+>
+> Follow instructions at <https://www.wireguard.com/install/> to install it.
+
+After ["Add an external node category"](#add-an-external-node-category), you could run:
+
+```sh
+sudo clever ng --ng 'first-ng' join --node-category-id 'dev-laptops-category' --label 'My laptop'
+```
+
+```log
+External peer 'external_2dfac7bf-ebab-4bf7-9763-d83f465fbc86' must have been added to networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+Activated WireGuard® tunnel
+Successfully joined networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'
+^C
+External peer 'external_2dfac7bf-ebab-4bf7-9763-d83f465fbc86' must have been removed from networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- [Use `--interactive` to answer optional questions](#use---interactive-to-answer-optional-questions)
+- `-c` is a short alias for `--node-category-id`.
+- `--private-key` allows you to provide a custom WireGuard® private key if you need.
```

Comment:
Too many "you". 

 `--private-key` allows to provide a custom WireGuard® private key if needed.